packages/rspack/src/builtin-plugin/LightningCssMiminizerRspackPlugin.ts

```diff
@@ -37,7 +38,7 @@ export const LightningCssMinimizerRspackPlugin = create(
 	): RawLightningCssMinimizerRspackPluginOptions => {
 		const { include, exclude, draft, nonStandard, pseudoClasses } =
 			options?.minimizerOptions ?? {};
-		const targets = options?.minimizerOptions?.targets ?? "fully supports es6"; // last not support es module chrome version
+		const targets = options?.minimizerOptions?.targets ?? browserslist(null);
```

Comment:
If there is no browserslist config, I think we should not change the default "fully supports es6". It is a potentially breaking change.

Reply:
And we should browserslist config in the Rust side for better performance. Similar to https://github.com/web-infra-dev/rspack/pull/7544